bpfd/src/multiprog/xdp.rs

```diff
@@ -75,11 +77,32 @@ impl XdpDispatcher {
             None,
             None,
         );
-        let (path, section_name) = image
-            .get_image(None)
+        let (tx, rx) = oneshot::channel();
```

Comment:
I think we should consider pre-pulling the dispatcher images at bpfd startup rather than attempting to every time Dispatcher::new is called

but we can do that in a follow up.